arkouda/pdarrayclass.py

```diff
@@ -396,63 +470,79 @@ def __setitem__(self, key, value):
             raise TypeError("Unhandled key type: {} ({})".\
                             format(key, type(key)))
 
-    def fill(self, value : object) -> None:
+    def fill(self, value : 'pdarray') -> None:
         """
         Fill the array (in place) with a constant value.
         """
         generic_msg("set {} {} {}".format(self.name, 
                                         self.dtype.name, self.format_other(value)))
 
-    def any(self):
+    def any(self) -> bool:
         """
         Return True iff any element of the array evaluates to True.
         """
         return any(self)
 
-    def all(self):
+    def all(self) -> bool:
         """
         Return True iff all elements of the array evaluate to True.
         """
         return all(self)
 
-    def is_sorted(self):
+    def is_sorted(self) -> bool:
         """
         Return True iff the array is monotonically non-decreasing.
+        
+        Parameters
+        ----------
+        None
+        
+        Returns
+        -------
+        bool 
+            Indicates if the array is monotonically non-decreasing
+            
+        Raises
+        ------
+        TypeError
+            Raised if pda is not a pdarray instance
+        RuntimeError
+            Raised if there's a server-side error thrown
         """
         return is_sorted(self)
 
-    def sum(self):
+    def sum(self) -> float:
         """
         Return the sum of all elements in the array.
         """
         return sum(self)
 
-    def prod(self):
+    def prod(self) -> float:
         """
         Return the product of all elements in the array. Return value is
         always a float.
         """
         return prod(self)
 
-    def min(self):
+    def min(self) -> float:
         """
         Return the minimum value of the array.
         """
         return min(self)
 
-    def max(self):
+    def max(self) -> float:
         """
         Return the maximum value of the array.
         """
         return max(self)
 
-    def argmin(self):
+    def argmin(self) -> float:
```

Comment:
return is an int64

Reply:
Yeah, the words "Return the index of " strongly imply that, LOL. Thanks for the heads-up, fixed in next checkin

Reply:
fixed!